nntrainer/include/optimizer.h

```diff
@@ -153,6 +153,17 @@ class Optimizer {
    */
   int initialize(TensorDim d, bool setTensor);
 
+  /**
+   * @brief     initialize optimizer. Initialize Weight if it is adam
+   * @param[in] list_d TensorDim list
```

Comment:
Lets add some comment on the order of elements this list should maintain (like W0, B0, W1, B1 if applicable etc)

Reply:
done.